test/generateTasks.spec.js

```diff
@@ -102,15 +101,18 @@ describe('generateTasks', () => {
   })
 
   it('should not match non-children files', () => {
-    const relPath = path.join(process.cwd(), '..')
-    resolveGitDir.mockReturnValueOnce(relPath)
+    /* eslint-disable */
```

Comment:
Why is eslint needs to be disabled?

Reply:
If unavoidable, please add particular rule which needs to be disabled and re-enable after the line which  needs to be avoided. Ideally, we should not used this.